tests/unit/src/test/scala/tests/SyntaxErrorSlowSuite.scala

```diff
@@ -312,4 +312,26 @@ object SyntaxErrorSlowSuite extends BaseSlowSuite("syntax-error") {
     )
   )
 
+  testAsync("literal-types") {
+    cleanWorkspace()
+    for {
+      _ <- server.initialize(
+        s"""
+           |/metals.json
+           |{"a": {}}
+           |/project/build.properties
+           |sbt.version=1.2.6
+           |/build.sbt
```

Comment:
Let's remove all build-related files, we only need a standalone file to test this change.

Side note, tests with sbt builds should live in the `slow` project.

Reply:
👍 I've added those info for debugging, then I forgot to remove them.

Reply:
ah, the standalone file reports an error (from scalac, not scalameta) probably because "rambo mode" uses 2.12. Should we change it to use 2.13 instead? This is not strictly related to this issue, again, but it might make sense

Reply:
Can you elaborate? The standalone compiler doesn't report diagnostics.

The "rambo mode" can stay on 2.12 for now, we can change it to 2.13 when the metals server upgrades to 2.13.

Reply:
Sorry, I got confused. :D Tests should be ok now.